server/pages/index.jade

```diff
@@ -54,6 +54,8 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				link(rel='stylesheet', href=urls['style-debug.css'])
 			else
 				link(rel='stylesheet', href=urls['style.css'])
+		style(type='text/css' id='critical-path-css')
+			!= Array.from(css).join('')
```

Comment:
Currently, we are including the styles of all **mounted** components which uses the `withStyles` HOC.

The way this HOC works is that when we render a page on the server, `insertCss` is called [for each component](https://github.com/kriasoft/isomorphic-style-loader/blob/master/src/withStyles.js#L22) and the styles ([`as text`](https://github.com/kriasoft/isomorphic-style-loader/blob/master/src/index.js#L30)) are added to the `css` Set.
Once all component are rendered we have a "critical path CSS" as a set of styles.

We can always improve it by customizing the `insertCss` function later, but I think the default behavior is already pretty good.